src/v2/spec.yaml

```diff
@@ -3357,14 +3356,20 @@ components:
         subscription:
           $ref: "#/components/schemas/WebhookSubscriptionFilters"
     WebhookPutReqBody:
-      allOf:
```

Comment:
Semi-related: this reminds me of how `readOnly` and `writeOnly` properties could enable us to use the same types for responses and requests. Might allow us to collapse some of these body types together.

https://swagger.io/docs/specification/v3_0/data-models/data-types/#:~:text=Read%2DOnly%20and,not%20in%20responses.